js2.js

```diff
@@ -44,6 +67,15 @@ Return: [2, 4, 6]
 */
 
 //Concept 5
+function myFunctionFive(arr){
+    var newArray = [];
+    arr.forEach(element => {
```

Comment:
Check out the challenge in the commit comments. Try it if you’d like, or you can skip it.